options.rb

```diff
@@ -0,0 +1,82 @@
+class Options
+  def initialize(app)
+    @app = app
+  end
+
+  def display
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def list_all_books
+    puts 'Books:-'
+    @app.list_all_books
+  end
+
+  def list_all_peoples
```

Comment:
**You did great job your app is working as expected if you fix this issue**

- [ ] You have grammar issue. I do not thing there is a word peoples :) kindly change all `list_all_peoples` to `list_all_people`. Because it's causing an issue when I run the app and, click `2 - List all people`